src/app/landingPage/termsCondition/page.tsx

```diff
@@ -3,96 +3,85 @@ import styles from '@/styles/privacy.module.scss';
-          materials may contain inaccuracies or errors and we expressly exclude
-          liability for any such inaccuracies or errors to the fullest extent
-          permitted by law.
-        </li>
-        <li className={styles.term_points}>
-          Your use of any information or materials on this website is entirely
-          at your own risk, for which we shall not be liable. It shall be your
-          own responsibility to ensure that any products, services or
-          information available through this website meet your specific
-          requirements.
-        </li>
-        <li className={styles.term_points}>
-          This website contains material which is owned by or licensed to us.
-          This material includes, but is not limited to, the design, layout,
-          look, appearance and graphics. Reproduction is prohibited other than
-          in accordance with the copyright notice, which forms part of these
-          terms and conditions.
-        </li>
-        <li className={styles.term_points}>
-          All trademarks reproduced in this website which are not the property
-          of, or licensed to, the operator are acknowledged on the website.
-        </li>
-        <li className={styles.term_points}>
-          Unauthorized use of this website may give rise to a claim for damages
-          and/or be a criminal offense.
-        </li>
-        <li className={styles.term_points}>
-          From time to time this website may also include links to other
-          websites. These links are provided for your convenience to provide
-          further information.
-        </li>
-        <li className={styles.term_points}>
-          You may not create a link to this website from another website or
-          document without TECHNOGETIC PVT LTD’s prior written consent.
-        </li>
-        <li className={styles.term_points}>
-          Your use of this website and any dispute arising out of such use of
-          the website is subject to the laws of India or other regulatory
-          authority.
-        </li>
-        <li className={styles.term_points}>
-          We as a merchant shall be under no liability whatsoever in respect of
-          any loss or damage arising directly or indirectly out of the decline
-          of authorization for any Transaction, on Account of the Cardholder
-          having exceeded the preset limit mutually agreed by us with our
-          acquiring bank from time to time
-        </li>
-      </ul>
-      <h3 className={styles.term_con_lastheading}>Cancellation & Refund Policy</h3>
-      <h3 className={styles.term_con_lastheading}>Last updated on Aug 1st 2023</h3>
-      <p>No cancellations & Refunds are entertained</p>
-  
-    </div>
+      <div className={styles.container2}>
+        <h1 className={styles.term_con_heading}>Terms & Conditions</h1>
+        <h2 className={styles.term_con_subheading}>Last updated on Aug 1st 2023</h2>
```

Comment:
is it possible to create a array of objects and map this data?

Reply:
done.